electrode-ota-server-routes-acquisition/src/index.js

```diff
@@ -1,48 +1,79 @@
-import { wrap, reqFields, streamToBuf } from 'electrode-ota-server-util';
+import { wrap, reqFields } from "electrode-ota-server-util";
 import diregister from "electrode-ota-server-diregister";
-const noContent = (reply) => (e) => {
-    if (e) return reply(e);
-    reply().code(204);
-};
-const ok = (reply) => (e) => {
+import keysToCamelOrSnake from "./keys-to-camel-or-snake";
+
+const HTTP_OK = 200;
+
+const ok = reply => e => {
     if (e) return reply(e);
-    reply('OK').code(200);
+    reply("OK").code(HTTP_OK);
 };
 
 export const register = diregister({
-    name: 'acquisitionRoute',
-    dependencies: ['electrode:route', 'ota!acquisition', 'ota!logger']
+    name: "acquisitionRoute",
+    dependencies: ["electrode:route", "ota!acquisition", "ota!logger"]
+// eslint-disable-next-line max-params
 }, (options, route, acquisition, logger) => {
     const {
         download,
         updateCheck,
         downloadReportStatus,
         deployReportStatus
-
     } = wrap(acquisition);
 
-    route([
+    const handleUpdateCheck = (request, reply) => {
+        logger.info(reqFields(request), "updateCheck request");
+        const snakeCaseRequested = /update_check/.test(request.url.path);
```

Comment:
you'll probably want to be more specific in this test.   `/updateCheck?update_check=true` will also be true.

Reply:
using`request.url.pathname` to determine the request action